src/modules/layout/components/UserTopNav.tsx

```diff
@@ -1,12 +1,11 @@
-import { transientOptions } from "@/modules/common/transientOptions";
 import styled from "@emotion/styled";
 import { User } from "@phosphor-icons/react";
 import { Button, theme } from "antd";
 import Image from "next/image";
 
 const UserTopNav = () => {
   const {
-    token: { colorPrimary, colorPrimaryBg },
+    token: { colorPrimary },
```

Comment:
Does we need this line can we use pattern like `${(props) => props.theme.antd.colorPrimary}` (asking)?